types/three/src/objects/InstancedMesh.d.ts

```diff
@@ -5,8 +5,11 @@ import { Mesh } from './Mesh';
 import { Matrix4 } from './../math/Matrix4';
 import { Color } from './../math/Color';
 
-export class InstancedMesh extends Mesh {
-    constructor(geometry: BufferGeometry, material: Material | Material[], count: number);
+export class InstancedMesh<
+    Geom extends BufferGeometry = BufferGeometry,
+    Mat extends Material | Material[] = Material
```

Comment:
I'm not a big fan of calling them `Geom` and `Mat` since those are just abbreviations of the original types. Can we stick with the old `TGeometry` and `TMaterial` to specify that they are type generics?